pkg/kubelet/kubelet.go

```diff
@@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
 	info, err := kl.GetCachedMachineInfo()
 	if err != nil {
-		glog.Errorf("error getting machine info: %v", err)
+		// TODO(roberthbailey): This is required for test-cmd.sh to pass.
```

Comment:
Does this remove the ability of the administrator to set arbitrary capacity to a node via the API (if the kubelet always writes 0)?

----- Original Message -----
> > @@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
> >  	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
> >  	info, err := kl.GetCachedMachineInfo()
> >  	if err != nil {
> > -		glog.Errorf("error getting machine info: %v", err)
> > +		// TODO(roberthbailey): This is required for test-cmd.sh to pass.
> 
> +1 to a zero capacity being a sane default. cAdvisor should work in test-cmd
> in Linux builds (unless we're injecting a fake).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6949/files#r30424260